packages/cli/src/commands/deploy.js

```diff
@@ -36,6 +36,9 @@ class DeployCommand extends Command {
     if (!path) {
       this.error("Please define your app path in your package.json or use init to create spaship.yaml ");
     }
+    if (!path.startsWith("/")) {
+      this.error("Path should be prefixed with `/`, got: " + path);
```

Comment:
Why not add a `/` and without throw error ?

Reply:
https://github.com/spaship/spaship/issues/432

Reply:
@kunyan  made the changes